.kokoro/github/ubuntu/gpu/build.sh

```diff
@@ -13,7 +13,7 @@ source venv/bin/activate
 python --version
 python3 --version
 
-export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:"
+#export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:"
```

Comment:
Oh, so this is what was causing the errors on JAX GPU?

Reply:
If so, remove this line or add a comment explaining

Reply:
Yes, detailed discussion here https://github.com/keras-team/keras-hub/pull/2252

Reply:
added comment